databuilder/extractor/generic_extractor.py

```diff
@@ -32,7 +32,7 @@ def init(self, conf):
 
             self._iter = iter(results)
         else:
-            raise RuntimeError('model class needs to be provided!')
+            self._iter = iter(self.values)
```

Comment:
what is this line about?

Reply:
Instead of instantiating in extractor, I am trying to pass list of model object to `generic_extractor`. Replacing this line to unblock it.